client/client.py

```diff
@@ -0,0 +1,52 @@
+import requests
+from .status import Status
+
+
+class Client:
+    """
+    Makes requests for the app
+    """
+
+    def __init__(self, url: str) -> None:
+        """
+        :param url: make requests to the url
+        """
+        self.url = url
+
+    def upload(self, file_path: str) -> str:
+        """
+        Make a request to upload a file to the app.
+        :param file_path: to a pptx file to upload.
+        :return: UID from the response.
+        :raise Exception: status code - when the response contains error code.
+        """
+        with open(file_path, 'rb') as f:
+            response = requests.post(
+                f"{self.url}/upload",
+                files={'file': f}
+            )
+        if response:  # True if the status code was between 200 and 400
+            return response.json()['uid']
+        raise Exception(response.status_code)
```

Comment:
It would be better to catch the exception than just raise it.
You may also tackle an I/O problem when opening the file.